MARKET_ADD.md

```diff
@@ -0,0 +1,88 @@
+- `collateralFactor` the percentage of value of supplied assets that is counted towards an account's collateral value, scaled up by `1e18`. Must be less than `1e18` to ensure system solvency.
+- `reserveFactor` percentage of interest accrued that goes to the Moonwell DAO reserves, scaled up by `1e18`.
+- `seizeShare` percentage of seized collateral that goes to protocol reserves when a liquidation occurs, scaled up by `1e18`.
+- `supplyCap` cap of amount of assets that can be supplied for a given market.
+- `borrowCap` cap of amount of assets that can be borrowed for a given market.
+- `priceFeedName` name of the chainlink aggregator address in `Addresses.sol`. **Note user must add the address to Addresses.sol on the proper network pre-deployment in order for the price feed to be set correctly**
+- `tokenAddressName` name of the underlying token address name for the new market, set in `Addresses.sol`. **Note user must add the underlying token address to Addresses.sol on the proper network pre-deployment in order for the MToken's underlying address to be set correctly**
+- `name` name of the Moonwell MToken. Prefixed with `Moonwell`.
+- `symbol` symbol of the Moonwell MToken. Prefixed with `m`.
+- `addressesString` name of the address string set in `Addresses.sol`.
+- `jrm` parameters for the configuration of a JumpRateModel for each MToken.
+- `baseRatePerYear` cost to borrow per year as a percentage, scaled by `1e18`.
+- `multiplierPerYear` multiplier on the base rate, which creates the curve of the rate before kink is hit, as a percentage, scaled by `1e18`.
+- `jumpMultiplierPerYear` rate multiplier as a percentage, scaled by `1e18` after kink is hit.
+- `kink` the point on the utilization curve after which the interest rate spikes using `jumpMultiplierPerYear` as a percentage, scaled by `1e18`
+
+## 2. Proposal Description
+
+Once the proposal description has been created, copy and paste it into [`MarketListingDescription.md`](./test/proposals/mips/examples/mip-market-listing/MarketListingDescription.md), deleting all previous data from this file.
+
+## 3. Environment Variables
+Once both the `MarketListingDescription.md` and `MTokens.json` file have the necessary contents, environment variables must be set for the script to read in their path. In the example folder, run these commands:
+
+```
+export LISTING_PATH="./test/proposals/mips/examples/mip-market-listing/MarketListingDescription.md"
+export MTOKENS_PATH="./test/proposals/mips/examples/mip-market-listing/MTokens.json"
+```
+If any errors show up relating to not being able to read in a file, double check the environment variables and make sure the paths are correct.
+
+## 4. Deployment
+To deploy these new markets, run `DeployMarketCreationProposal.s.sol` using command:
+
+```
+forge script test/proposals/DeployMarketCreationProposal.s.sol:DeployMarketCreationProposal \
```

Comment:
This worked great for me, but I didn't broadcast yet. I don't want to actually deploy contracts on mainnet yet, so there aren't multiple deploys with the same symbol.

I would like to be able to deploy on testnet, so we can see how the governance proposal works.

Reply:
you should be able to deploy on testnet by just changing the rpc url in that same command